.github/workflows/release_buid_test.yaml

```diff
@@ -0,0 +1,159 @@
+name: Release build test
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      build_type:
+        type: choice
+        description: "Memgraph Build type. Default value is Release."
+        default: 'Release'
+        options:
+          - Release
+          - RelWithDebInfo
+
+  push:
+    branches:
+      - "release/**"
+    tags:
+      - "v*.*.*-rc*"
+      - "v*.*-rc*"
+  schedule:
+    - cron: "0 22 * * *"
+
+env:
+  THREADS: 24
+  MEMGRAPH_ENTERPRISE_LICENSE: ${{ secrets.MEMGRAPH_ENTERPRISE_LICENSE }}
+  MEMGRAPH_ORGANIZATION_NAME: ${{ secrets.MEMGRAPH_ORGANIZATION_NAME }}
+  BUILD_TYPE: ${{ github.event.inputs.build_type || 'Release' }}
+
+jobs:
+  Debian10:
+    uses: ./.github/workflows/release_debian10.yaml
+    with:
+      build_type: ${{ github.event.inputs.build_type || 'Release' }}
+    secrets: inherit
+
+  Ubuntu20_04:
+    uses: ./.github/workflows/release_ubuntu2004.yaml
+    with:
+      build_type: ${{ github.event.inputs.build_type || 'Release' }}
+    secrets: inherit
+
+  PackageDebian10:
+    if: github.ref_type == 'tag'
+    needs: [Debian10]
+    runs-on: [self-hosted, DockerMgBuild, X64]
+    timeout-minutes: 60
+    steps:
+      - name: "Set up repository"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required because of release/get_version.py
+      - name: "Build package"
+        run: |
+          ./release/package/run.sh package debian-10 $BUILD_TYPE
+      - name: "Upload package"
+        uses: actions/upload-artifact@v4
+        with:
+          name: debian-10
+          path: build/output/debian-10/memgraph*.deb
+
+  PackageUbuntu20_04:
+    if: github.ref_type == 'tag'
+    needs: [Ubuntu20_04]
+    runs-on: [self-hosted, DockerMgBuild, X64]
+    timeout-minutes: 60
+    steps:
+      - name: "Set up repository"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required because of release/get_version.py
+      - name: "Build package"
+        run: |
+          ./release/package/run.sh package ubuntu-22.04 $BUILD_TYPE
+      - name: "Upload package"
+        uses: actions/upload-artifact@v4
+        with:
+          name: ubuntu-22.04
+          path: build/output/ubuntu-22.04/memgraph*.deb
+
+  PackageUbuntu20_04_ARM:
+    if: github.ref_type == 'tag'
+    needs: [Ubuntu20_04]
+    runs-on: [self-hosted, DockerMgBuild, ARM64]
+    timeout-minutes: 120
```

Comment:
Because of our ARM infrastructue, the M1 Mac (`strange`), can sometimes take longer than 1 hour to complete this job. The end product is always valid. 
I agree 2 hours is too long, how about we do 90 minutes?